backend/src/main/resources/application.properties

```diff
@@ -103,5 +103,16 @@ quarkus.unleash.active=false
 quarkus.unleash.url=http://localhost:4242
 
 # Kessel
-notifications.kessel.inventory-api.target-url=localhost:9080
-notifications.kessel.relationships-api.target-url=localhost:9000
+inventory-api.authn.client.id=insights-notifications
+inventory-api.authn.client.issuer=http://localhost:8084/realms/redhat-external
```

Comment:
Does this URL help with local testing? Or could we use `https://redhat.com/realms/redhat-external` here?

Reply:
Yes, it does help with local testing if you set up Keycloak locally, but... I'm not sure if we should change it to the URL you pointed :thinking:

Reply:
If it helps with local testing, then the current value is fine.